src/components/organisms/AssetActions/Consume.tsx

```diff
@@ -164,6 +167,7 @@ export default function Consume({
       <div className={styles.info}>
         <div className={styles.filewrapper}>
           <File file={file} />
+          <AssetStatus did={file.valid} />
```

Comment:
this is the confusing usage, cause why would the did be a boolean?

As for the typing errors I see that `valid` was added over on ocean.js so this PR needs to have that new ocean.js version bumped for the build and tests to succeed